R/summary-log.R

```diff
@@ -72,9 +73,20 @@ add_summary <- function(
 #' @importFrom tibble tibble
 #' @importFrom tidyselect all_of
 #' @param .mods List of model objects that will be passed to [model_summaries()].
+#' @param calc_aic_bic Logical(T/F). If `TRUE`, calculate the Akaike Information
+#' Criterion (AIC) and Bayesian Information Criterion (BIC)
 #' @param ... Arguments passed through to [model_summaries()]
+#'
+#' @details
+#' `calc_aic_bic` should be set to `FALSE` for certain model types, such as
+#'  bootstrap runs.
+#'
 #' @keywords internal
-summary_log_impl <- function(.mods, ...) {
+summary_log_impl <- function(
+    .mods,
+    calc_aic_bic = TRUE,
```

Comment:
This argument gets exposed to the user via `...`, which is documented as getting passed to `model_summaries`.

In [this message](https://github.com/metrumresearchgroup/bbr/pull/731#discussion_r1954944405), you say it's for "internal control only". In that case, you could change the internal spots to use `summary_log_impl` (e.g., `summarize_bootstrap_run`) and avoid exposing it externally (e.g., by adjusting `summary_log` and `add_summary` to pass `calc_aic_bic=TRUE` explicitly).

If the intention is to expose it, then `calc_aic_bic` should be added as an argument to `summary_log` and documented.